libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/MapboxNavigationOptions.java

```diff
@@ -51,12 +51,6 @@
 
   public abstract int locationAcceptableAccuracyInMetersThreshold();
 
-  public abstract int locationAccuracyPercentThreshold();
```

Comment:
Are we ok here breaking SemVer?
If so, we should document this in the release notes.